apps/service_define/test/define/view_state_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Define.ViewStateTest do
+  use ExUnit.Case
+  alias Define.ViewState
+
+  setup do
+    server = start_supervised!(ViewState)
+    {:ok, server: server}
+  end
+
+  test "when new_greeting event sets the greeting", %{ server: server } do
+    ui_state = ViewState.event(server, "new_greeting", %{"greeting" => "Hello World!"})
```

Comment:
what is this for?

Reply:
This test just proved the GenServer can receive events from the UI and update the UI state. The greeting stuff is just contrived for now as we needed something simple to prove the whole thing works. The greeting stuff will be removed once we have actual functionality here.